src/index.ts

```diff
@@ -16,14 +16,15 @@ export function $switchCaseStrict<K extends string | number, V>(
 }
 
 export function $switchCaseDefault<K extends string | number, V>(
-  input: K,
+  input: K | null | undefined,
   cases: { [k in K]?: () => V },
   defaultCase: () => V
 ): V {
-  const candidate = cases[input];
-  if (candidate) {
-    const value = candidate();
-    return value;
+  if (input) {
```

Comment:
this is too eager, an empty string would fail this check but it might be a valid key in an object